app/templates/Tags/view.php

```diff
@@ -0,0 +1,77 @@
+            <?= $this->Html->link(__('New Tag'), ['action' => 'add'], ['class' => 'side-nav-item']) ?>
+        </div>
+    </aside>
+    <div class="column-responsive column-80">
+        <div class="tags view content">
+            <h3><?= h($tag->title) ?></h3>
+            <table>
+                <tr>
+                    <th><?= __('Title') ?></th>
+                    <td><?= h($tag->title) ?></td>
+                </tr>
+                <tr>
+                    <th><?= __('Id') ?></th>
+                    <td><?= $this->Number->format($tag->id) ?></td>
+                </tr>
+                <tr>
+                    <th><?= __('Created') ?></th>
+                    <td><?= h($tag->created) ?></td>
+                </tr>
+                <tr>
+                    <th><?= __('Modified') ?></th>
+                    <td><?= h($tag->modified) ?></td>
+                </tr>
+            </table>
+            <div class="related">
+                <h4><?= __('Related Articles') ?></h4>
+                <?php if (!empty($tag->articles)) : ?>
+                <div class="table-responsive">
+                    <table>
+                        <tr>
+                            <th><?= __('Id') ?></th>
+                            <th><?= __('User Id') ?></th>
+                            <th><?= __('Title') ?></th>
+                            <th><?= __('Slug') ?></th>
+                            <th><?= __('Body') ?></th>
+                            <th><?= __('Published') ?></th>
+                            <th><?= __('Created') ?></th>
+                            <th><?= __('Modified') ?></th>
+                            <th class="actions"><?= __('Actions') ?></th>
+                        </tr>
+                        <?php foreach ($tag->articles as $articles) : ?>
+                        <tr>
+                            <td><?= h($articles->id) ?></td>
+                            <td><?= h($articles->user_id) ?></td>
+                            <td><?= h($articles->title) ?></td>
+                            <td><?= h($articles->slug) ?></td>
+                            <td><?= h($articles->body) ?></td>
+                            <td><?= h($articles->published) ?></td>
+                            <td><?= h($articles->created) ?></td>
+                            <td><?= h($articles->modified) ?></td>
+                            <td class="actions">
+                                <?= $this->Html->link(__('View'), ['controller' => 'Articles', 'action' => 'view', $articles->id]) ?>
+                                <?= $this->Html->link(__('Edit'), ['controller' => 'Articles', 'action' => 'edit', $articles->id]) ?>
+                                <?= $this->Form->postLink(__('Delete'), ['controller' => 'Articles', 'action' => 'delete', $articles->id], ['confirm' => __('Are you sure you want to delete # {0}?', $articles->id)]) ?>
```

Comment:
```suggestion
                                <?= $this->Html->link(__('View'), ['controller' => 'Articles', 'action' => 'view', $articles->slug]) ?>
                                <?= $this->Html->link(__('Edit'), ['controller' => 'Articles', 'action' => 'edit', $articles->slug]) ?>
                                <?= $this->Form->postLink(__('Delete'), ['controller' => 'Articles', 'action' => 'delete', $articles->id], ['confirm' => __('Are you sure you want to delete # {0}?', $articles->slug)]) ?>
```




Refer

https://github.com/ArielGalvez/tutorial-cakephp/blob/98358657d46647b0ec6fb4f87382c72f59ea5d7b/app/src/Controller/ArticlesController.php#L47

https://github.com/ArielGalvez/tutorial-cakephp/blob/98358657d46647b0ec6fb4f87382c72f59ea5d7b/app/src/Controller/ArticlesController.php#L92

https://github.com/ArielGalvez/tutorial-cakephp/blob/98358657d46647b0ec6fb4f87382c72f59ea5d7b/app/src/Controller/ArticlesController.php#L126

All 3 action (view, edit, delete) use slug instead of id. When accessing with id we'll get error record not found due to we pass the wrong value. We shall pass slug instead of id.

Please check.

Reply:
changed to slug instead of id